index.html

```diff
@@ -0,0 +1,103 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Backgrounds and Gradients</title>
+  </head>
+  <body>
+    <header>
+      <nav class="hea-foo bg-gray">
+        <ul class="height100">
+          <li class="apple-logo">
+            <a href="">
+              <p></p>
+            </a>
+          </li>
+          <li>
+            <a href="">Store</a>
+          </li>
+          <li>
+            <a href="">Mac</a>
+          </li>
+          <li>
+            <a href="">iPod</a>
+          </li>
+          <li>
+            <a href="">iPhone</a>
+          </li>
+          <li>
+            <a href="">iPad</a>
+          </li>
+          <li>
+            <a href="">iTunes</a>
+          </li>
+          <li>
+            <a href="">Support</a>
+          </li>
+          <li id="search-bar">
+            <input type="search" class="width80" /><a href=""></a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="width100">
+      <div id="verse1" class="position-relative width70 height90 float-left">
+        <a href="#"></a>
+      </div>
+      <div id="verse2" class="position-relative width80 height90 float-left">
+        <a href="#"></a>
+      </div>
+      <div id="verse3" class="position-relative width70 float-left">
+        <a href="#"></a>
+      </div>
+    </main>
+    <section class= "position-relative width100">
+      <h1> this is hidden </h1>
+	  <ul>
+        <li id="s-5" class="float-left">
+          <a href="#"></a>
+        </li>
+        <li id="c-5" class="float-left">
+          <a href="#"></a>
+        </li>
+        <li id="years-30" class="float-left">
+          <a href="#"></a>
+        </li>
+        <li id="supplier" class="float-left">
+          <a href="#"></a>
+        </li>
+      </ul>
+    </section>
```

Comment:
[SUGGESTION] Maybe the `<aside>` tag would fit better this part of the project.

> The <aside> element is used to identify content that is related to the primary content of the webpage, but does not constitute the primary content of the page. Author information, related links, related content, and advertisements are exampes of content that may be found in an aside element.


[Read more](https://html.com/tags/aside/#ixzz5yaXtnPH6)